packages/flutter_tools/lib/src/resident_runner.dart

```diff
@@ -132,7 +132,7 @@ class FlutterDevice {
   final Device device;
   final ResidentCompiler generator;
   Stream<Uri> observatoryUris;
-  List<VMService> vmServices;
```

Comment:
Maybe this cleanup should go in a separate PR.

Reply:
Removed